.github/workflows/build-operator-image.yaml

```diff
@@ -13,15 +13,16 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build Image
-        run: |
-          IMG=eda-server-operator:main make docker-build
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
 
-      - name: Push To Quay
-        uses: redhat-actions/push-to-registry@v2.1.1
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
         with:
-          image: eda-server-operator
-          tags: main
-          registry: quay.io/ansible/
+          registry: quay.io
           username: ${{ secrets.QUAY_USER }}
           password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and Push Image
+        run: |
+          docker buildx build --push --platform=linux/arm64,linux/amd64,linux/s390x,linux/ppc64le --tag quay.io/ansible/eda-server-operator:main -f Dockerfile .
```

Comment:
for consistency with comment https://github.com/ansible/eda-server-operator/pull/159#discussion_r1444377284

``` suggestion
      - name: Log into registry ghcr.io
        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
        with:
          registry: ghcr.io
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}


      - name: Log into registry quay.io
        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
        with:
          registry: quay.io/ansible/
          username: ${{ secrets.QUAY_USER }}
          password: ${{ secrets.QUAY_TOKEN }}


      - name: Build and Store Image @ghcr
        run: |
          IMG=ghcr.io/${{ github.repository_owner }}/eda-server-operator:${{ github.sha }} make docker-buildx


      - name: Publish Image to quay.io/eda-server-operator:main
        run: |
          docker buildx imagetools create ghcr.io/${{ github.repository_owner }}/eda-server-operator:${{ github.sha }} --tag quay.io/ansible/eda-server-operator:main
```

Reply:
gah, I made a mess by committing another change first, so this commit suggestion wouldn't apply via the UI.  I added this commit and credited it to you in the commit message.  Thanks for this suggestion!
* https://github.com/ansible/eda-server-operator/pull/161/commits/956957d57ea2ca4fc8d5d26fd32feb0b59a31f45